javatests/arcs/android/entity/DifferentHandleManagerTest.kt

```diff
@@ -63,7 +62,7 @@ class DifferentHandleManagerTest : HandleManagerTestBase() {
   override fun tearDown() {
     super.tearDown()
     runBlocking {
-      stores.reset()
+      storageEndpointManager.reset()
```

Comment:
Should the coroutine scope passed to the storageEndpointManager also be canceled here?

Reply:
Yes, I think so.